db/migrate/20151015184210_create_index.rb

```diff
@@ -0,0 +1,5 @@
+class CreateIndex < ActiveRecord::Migration
+  def change
+    add_index :interactions, [:sentence, :keyword1, :keyword2], name: 'index_interactions'
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.